SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2392,7 +2389,14 @@ void USpatialReceiver::ResolveIncomingOperations(UObject* Object, const FUnrealO
 			continue;
 		}
 
-		FSpatialObjectRepState* RepState = DependentChannel->ObjectReferenceMap.Find(ChannelObjectIter->Value);
+		ObjectsToInspect.Push({ DependentChannel, ChannelObjectIter->Value });
+	}
+
+	for (const auto& ChannelObjectPair : ObjectsToInspect)
+	{
+		USpatialActorChannel* DependentChannel = ChannelObjectPair.Channel;
+		UObject* ReplicatingObject = ChannelObjectPair.Object.Get();
```

Comment:
do we want to check Object is valid at this point?

Reply:
I think we are safe at this point, since we check if the WeakObjectPtr is valid in the loop above and we couldn't have lost it from that. What could happen though, I guess, is that the object gets deleted in a repNotify call...

Reply:
worth check'ing ?

Reply:
Changed the code, and it now includes an if-continue for this